racingcar-java/src/main/java/RacingCar.java

```diff
@@ -0,0 +1,36 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RacingCar {
+
+    List<Car> cars;
+
+    public RacingCar(int numberCars) {
+        this.cars = setCars(numberCars);
+    }
+
+    List<Car> setCars(int numberCar) {
+        List<Car> cars = new ArrayList<Car>();
```

Comment:
```suggestion
        List<Car> cars = new ArrayList<>();
```
굳이 명시하지 않아도 되는 부분입니다 :)